stdlib/2and3/mimetypes.pyi

```diff
@@ -1,10 +1,16 @@
 # Stubs for mimetypes
 
-from typing import Dict, IO, List, Optional, Sequence, Text, Tuple
+from typing import Dict, IO, List, Optional, Sequence, Text, Tuple, AnyStr, Union
 import sys
 
-def guess_type(url: Text,
-               strict: bool = ...) -> Tuple[Optional[str], Optional[str]]: ...
+if sys.version_info >= (3, 8):
+    from os import PathLike
+    def guess_type(url: Union[Text, PathLike[AnyStr]],
```

Comment:
AnyStr doesn't make sense here because there is no other use of the typevar. Also, from my experiments it doesn't actually support `PathLike[bytes]`, so you can just put `PathLike[str]` (no need for `Text` as this is a Python 3-only branch).